eo-runtime/src/test/java/JulToSlf4jExtension.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import static org.junit.jupiter.api.extension.ExtensionContext.Namespace.GLOBAL;
+
+public class JulToSlf4jExtension implements BeforeAllCallback {
```

Comment:
@marceloamadeu here Javadoc is required (see how it's done for other classes)

Reply:
@yegor256 was added. Please verify.